pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -717,7 +717,7 @@ func (asw *actualStateOfWorld) AddPodToVolume(markVolumeOpts operationexecutor.M
 		// Update uncertain volumes - the new markVolumeOpts may have updated information.
 		// Especially reconstructed volumes (marked as uncertain during reconstruction) need
 		// an update.
-		updateUncertainVolume = utilfeature.DefaultFeatureGate.Enabled(features.SELinuxMountReadWriteOncePod) && podObj.volumeMountStateForPod == operationexecutor.VolumeMountUncertain
+		updateUncertainVolume = utilfeature.DefaultFeatureGate.Enabled(features.NewVolumeManagerReconstruction) && podObj.volumeMountStateForPod == operationexecutor.VolumeMountUncertain
```

Comment:
features.NewVolumeManagerReconstruction is promoted to GA and lock to default.

```suggestion
		updateUncertainVolume = podObj.volumeMountStateForPod == operationexecutor.VolumeMountUncertain
```

Reply:
Do we want to cherry-pick it into 1.27, 1.28, 1.29 and 1.30?  If so, please don't accept my suggestion on code changes and the current change makes senses

Reply:
Yeah, I just wanted to fix 1.27-1.29, as you mentioned, 1.30 is locked.

Reply:
Hi, @bertinatto Can you help to cherry-pick this PR into the 1.27, 1.28, and 1.29 release branches?

Reply:
Hi @carlory; sure, here are the cherry-picks, PTAL:

- 1.27: https://github.com/kubernetes/kubernetes/pull/124142
- 1.28: https://github.com/kubernetes/kubernetes/pull/124141
- 1.29: https://github.com/kubernetes/kubernetes/pull/124140

Reply:
Thank you. 😃